build-tools/trim-analyzers/trim-analyzers.props

```diff
@@ -0,0 +1,53 @@
+<!-- Import this to enable trim warnings of all kinds -->
+<Project>
+  <PropertyGroup>
+    <!-- Sets assembly metadata, enable analyzers -->
+    <IsTrimmable>true</IsTrimmable>
+    <EnableSingleFileAnalyzer>true</EnableSingleFileAnalyzer>
+    <EnableAotAnalyzer>true</EnableAotAnalyzer>
+    <!-- In app projects, tells ILLink to emit warnings as errors -->
+    <ILLinkTreatWarningsAsErrors>true</ILLinkTreatWarningsAsErrors>
+    <!--
+      Trim warnings, codes listed here:
+      * https://github.com/dotnet/runtime/blob/7403a062960d092c73ce3f07d3ff323ffdf7de43/src/tools/illink/src/linker/Resources/Strings.resx
+      * https://github.com/dotnet/docs/tree/9cb45cf9cd34f3b7259a023c3d92a124a87090d5/docs/core/deploying/trimming/trim-warnings
+    -->
+    <WarningsAsErrors>
+      $(WarningsAsErrors);
+      IL2000;IL2001;IL2002;IL2003;IL2004;
+      IL2005;IL2006;IL2007;IL2008;IL2009;
+      IL2010;IL2011;IL2012;IL2013;IL2014;
+      IL2015;IL2016;IL2017;IL2018;IL2019;
+      IL2020;IL2021;IL2022;IL2023;IL2024;
+      IL2025;IL2026;IL2027;IL2028;IL2029;
+      IL2030;IL2031;IL2032;IL2033;IL2034;
+      IL2035;IL2036;IL2037;IL2038;IL2039;
+      IL2040;IL2041;IL2042;IL2043;IL2044;
+      IL2045;IL2046;IL2047;IL2048;IL2049;
+      IL2050;IL2051;IL2052;IL2053;IL2054;
+      IL2055;IL2056;IL2057;IL2058;IL2059;
+      IL2060;IL2061;IL2062;IL2063;IL2064;
+      IL2065;IL2066;IL2067;IL2068;IL2069;
+      IL2070;IL2071;IL2072;IL2073;IL2074;
+      IL2075;IL2076;IL2077;IL2078;IL2079;
+      IL2080;IL2081;IL2082;IL2083;IL2084;
+      IL2085;IL2086;IL2087;IL2088;IL2089;
+      IL2090;IL2091;IL2092;IL2093;IL2094;
+      IL2095;IL2096;IL2097;IL2098;IL2099;
+      IL2100;IL2101;IL2102;IL2103;IL2104;
+      IL2105;IL2106;IL2107;IL2108;IL2109;
+      IL2110;IL2111;IL2112;IL2113;IL2114;
+      IL2115;IL2116;IL2117;IL2118;IL2119;
+      IL2120;IL2121;IL2122;IL2123;IL2124;
+      IL2125;IL2126;IL2127;IL2128;IL2129;
+    </WarningsAsErrors>
+    <!--
+      NativeAOT warnings, codes listed here:
+      * https://github.com/dotnet/docs/tree/9cb45cf9cd34f3b7259a023c3d92a124a87090d5/docs/core/deploying/native-aot/warnings
+    -->
+    <WarningsAsErrors>
```

Comment:
Recently I added `IlcTreatWarningsAsErrors` - for symmetry with the trimmer. https://github.com/dotnet/runtime/pull/96567

Reply:
I added this one for the future. 👍